package.json

```diff
@@ -37,7 +41,6 @@
     "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-react": "^7.31.11",
     "eslint-plugin-react-hooks": "^4.6.0",
-    "eslint-plugin-tailwindcss": "^3.7.1",
```

Comment:
Also remove usage in `.eslintrc.cjs` or eslint will not working well.

Reply:
![Screenshot 2023-01-10 133803](https://user-images.githubusercontent.com/18546096/211470315-f1461201-c9a0-4114-853d-9d127b7d5b99.png)